docs/core/reference/cored.md

```diff
@@ -0,0 +1,96 @@
+# cored Command
+
+Command `cored` provides the Chain Core daemon and API server.
+
+## Installation
+
+`cored` is included with all desktop installations of Chain Core
+Developer Edition, and is run by default when starting Mac, Windows and Docker.
+
+### From source
```

Comment:
Do we really want to describe building from source here? Is this document for users of cored or for developers of cored?

I feel like it might be better to keep this file focused on users' needs and leave out all mention of how to build cored. (Building is something only we have to worry about. That's internal documentation.)